### PR TITLE
Add TiPrometheus(Prometheus adapter for TiKV)

### PR DESCRIPTION
**Pull request by 70data**
Prometheus adapter for TiKV.
TiPrometheus support remote read and write.
TiKV is distributed transactional key-value database, originally created to complement TiDB.
I hope it solves the prometheus's storage problem.
I hope to provide a new idea for time series data storage.

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ data volumes.
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
   * [VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics): write
   * [Wavefront](https://github.com/wavefrontHQ/prometheus-storage-adapter): write
+  * [TiPrometheus](https://github.com/bragfoo/TiPrometheus): read and write
```
Comment (tomwilkie) [inline content/docs/operating/integrations.md]: Please keep the list sorted.

**Comment by tomwilkie** [comment]
You need to add a DCO to the existing commit.

**Comment by 70data** [comment]
Done.
Please check.
@tomwilkie

**Comment by aixeshunter** [comment]
Is there no unit test in `https://github.com/bragfoo/TiPrometheus`?
And no `Dockerfile`, `Makefile` there.
Could u please update them?

**Comment by 70data** [comment]
It's only a first edition but it's ready for production.
This will be followed by development, optimization of performance and availability.
And provide unit test、benchmark report、dockerfile and CI/CD tools.

**Comment by aixeshunter** [comment]
> It's only a first edition but it's ready for production.
> This will be followed by development, optimization of performance and availability.
> And provide unit test、benchmark report、dockerfile and CI/CD tools.
> Because tikv is a dynamically scalable kv storage, Prometheus is given the potential for unlimited expansion.
> I will continue to do it later.

OK, thx

File: content/docs/operating/integrations.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ data volumes.
   * [OpenTSDB](https://github.com/prometheus/prometheus/tree/master/documentation/examples/remote_storage/remote_storage_adapter): write
   * [PostgreSQL/TimescaleDB](https://github.com/timescale/prometheus-postgresql-adapter): read and write
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
+  * [TiPrometheus](https://github.com/bragfoo/TiPrometheus): read and write
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: This should be TiKV

Comment (70data) [reply content/docs/operating/integrations.md]: > This should be TiKV

Recovered.

**Pull request merged**